mdtk/pytorch_trainers.py

```diff
@@ -1,5 +1,6 @@
 import sys
 import warnings
+from collections import defaultdict
```

Comment:
What's the benefit of this over a normal dict?

Reply:
Oh, I read up on it. Essentially, it allows dict[i] += 1 without requiring a None check. Cool!